modules/channel-web/src/views/lite/main.tsx

```diff
@@ -230,6 +242,10 @@ class Web extends React.Component<MainProps> {
     }, constants.MIN_TIME_BETWEEN_SOUNDS)
   }
 
+  isLazySocket() {
+    return this.config.lazySocket || this.props.botInfo?.lazySocket
```

Comment:
If the host page configuration should prevail over the configuration on the studio, then it should use `this.config.lazySocket` only if it's not `undefined`, otherwise a `false` value would not be taken into account